src/command/feed/feed-command.ts

```diff
@@ -0,0 +1,98 @@
+import { Topic } from '@ethersphere/bee-js/dist/feed/topic'
+import Wallet from 'ethereumjs-wallet'
+import { Option } from 'furious-commander'
+import { bold, green } from 'kleur'
+import { exit } from 'process'
+import { getWalletFromIdentity } from '../../service/identity'
+import { Identity } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+import { VerbosityLevel } from '../root-command/command-log'
+
+export class FeedCommand extends RootCommand {
+  @Option({ key: 'identity', describe: 'Name of the identity', required: true })
+  public identity!: string
+
+  @Option({
+    key: 'topic',
+    describe: 'Feed topic',
+    default: '0'.repeat(64),
+    defaultDescription: '32 zero bytes',
+  })
+  public topic!: string
+
+  @Option({ key: 'password', describe: 'Password for the wallet' })
+  public password!: string
+
+  @Option({ key: 'hash-topic', type: 'boolean', describe: 'Hash the topic to 32 bytes', default: false })
+  public hashTopic!: boolean
+
+  protected async updateFeedAndPrint(chunkReference: string): Promise<void> {
+    this.console.dim('Updating feed...')
+    const wallet = await this.getWallet()
+    const topic = this.getTopic()
+    const writer = this.bee.makeFeedWriter('sequence', topic, wallet.getPrivateKey())
+    const { reference } = await writer.upload(chunkReference)
+    const manifest = await this.bee.createFeedManifest('sequence', topic, wallet.getAddressString())
+
+    if (this.verbosity === VerbosityLevel.Verbose) {
+      this.console.log(bold(`Chunk Reference -> ${green(chunkReference)}`))
+      this.console.log(bold(`Chunk Reference URL -> ${green(`${this.beeApiUrl}/files/${chunkReference}`)}`))
+      this.console.log(bold(`Feed Reference -> ${green(reference)}`))
+      this.console.log(bold(`Feed Manifest -> ${green(manifest)}`))
+    }
+    this.console.log(bold(`Feed Manifest URL -> ${green(`${this.beeApiUrl}/bzz/${manifest}/`)}`))
+
+    if (this.verbosity === VerbosityLevel.Quiet) {
+      this.console.important(manifest)
+    }
```

Comment:
Maybe the similar way as I mentioned in my earlier comment, you can define a neutral `quiet` function for the console messages in `command-log.ts`